.gitignore

```diff
@@ -8,6 +8,10 @@ package-lock.json
 .env.*.local
 .env.local
 
+
+### JetBrains ###
+.idea
+
```

Comment:
You can also edit the auto-generated selection at https://www.toptal.com/developers/gitignore?templates=node,linux,macos,windows , add JetBrains then replace this auto-generated block